data/sql/updates/pending_db_world/reputation_spillover_template-update.sql

```diff
@@ -0,0 +1,75 @@
+-- --------------------------------------------------------
+-- Värd:                         127.0.0.1
+-- Serverversion:                8.0.28 - MySQL Community Server - GPL
+-- Server-OS:                    Win64
+-- HeidiSQL Version:             11.3.0.6295
+-- --------------------------------------------------------
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET NAMES utf8 */;
+/*!50503 SET NAMES utf8mb4 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+-- Dumpar struktur för tabell acore_world.reputation_spillover_template
+DROP TABLE IF EXISTS `reputation_spillover_template`;
+CREATE TABLE IF NOT EXISTS `reputation_spillover_template` (
+  `faction` SMALLINT UNSIGNED NOT NULL DEFAULT '0' COMMENT 'faction entry',
+  `faction1` SMALLINT UNSIGNED NOT NULL DEFAULT '0' COMMENT 'faction to give spillover for',
+  `rate_1` FLOAT NOT NULL DEFAULT '0' COMMENT 'the given rep points * rate',
+  `rank_1` TINYINT UNSIGNED NOT NULL DEFAULT '0' COMMENT 'max rank,above this will not give any spillover',
+  `faction2` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
+  `rate_2` FLOAT NOT NULL DEFAULT '0',
+  `rank_2` TINYINT UNSIGNED NOT NULL DEFAULT '0',
+  `faction3` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
+  `rate_3` FLOAT NOT NULL DEFAULT '0',
+  `rank_3` TINYINT UNSIGNED NOT NULL DEFAULT '0',
+  `faction4` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
+  `rate_4` FLOAT NOT NULL DEFAULT '0',
+  `rank_4` TINYINT UNSIGNED NOT NULL DEFAULT '0',
+  `faction5` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
+  `rate_5` FLOAT NOT NULL DEFAULT '0',
+  `rank_5` TINYINT UNSIGNED NOT NULL DEFAULT '0',
+  `faction6` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
+  `rate_6` FLOAT NOT NULL DEFAULT '0',
+  `rank_6` TINYINT UNSIGNED NOT NULL DEFAULT '0',
+  PRIMARY KEY (`faction`)
+) ENGINE=MYISAM DEFAULT CHARSET=utf8mb4 ROW_FORMAT=DYNAMIC COMMENT='Reputation spillover reputation gain';
```

Comment:
Why are you not altering the table instead of just recreating it?

Reply:
Hmm, never thought about it, just went with the flow, should be resolved now ;].